### ISSUE TITLE
Draw bitmaps and customized labels on the action buttons, howto

### ISSUE THREAD

**Issue by q4osteam** [seb3773/actioncenter-kicker-applet-legacy#3]
Since button bitmaps are localized, we need to draw them at runtime. We provide here a simple example on how to draw customized button label.

A way is to create own button class derived from TQPushButton and use TQPainter to paint texts and images on the button, mybutton.cpp source is the clue. Compile the example:
$ tqmake base.pro
$ make

TQT3 assistant application is the perfect TQT classes documentation, install package "tqt3-assistant" to get it.

Find a variety of nice TQT coding examples in /usr/share/doc/tqt3-examples dir, you can easily compile all the examples and see how they work. Install "tqt3-examples" package.
[simpleapp1.tar.gz](https://github.com/user-attachments/files/17599960/simpleapp1.tar.gz)

**Comment by seb3773** [comment]
You're right, drawing buttons at runtime is the way to go, thank you for pointing me in the right direction, I will try to improve the applet with the help of your example :)
